src/e2e-test/features/bigquery/source/BigQuerySourceError.feature

```diff
@@ -55,3 +55,24 @@ Feature: BigQuery source - Validate BigQuery source plugin error scenarios
     Then Enter BigQuery source property table name
     Then Enter BigQuery property temporary bucket name "bqInvalidTemporaryBucket"
     Then Verify the BigQuery validation error message for invalid property "bucket"
+
+  @BQ_SOURCE_TEST
+  Scenario Outline:Verify if invalid Partition Start date and Partition end Date format at design time or check pipeline fails while preview
```

Comment:
Change the scenario title as per the scenario, here we are not checking at preview.